src/components/Tool/Appeal/InitialPage/AppealsInitialPage.tsx

```diff
@@ -12,13 +12,10 @@ import { useTranslation } from 'react-i18next';
 import { makeStyles } from 'tss-react/mui';
 import AddAppealForm from 'src/components/Tool/Appeal/InitialPage/AddAppealForm/AddAppealForm';
 import Appeals from 'src/components/Tool/Appeal/InitialPage/Appeals';
+import { ToolsGridContainer } from 'src/components/Tool/styledComponents';
 import { useAccountListId } from 'src/hooks/useAccountListId';
 
 const useStyles = makeStyles()((theme: Theme) => ({
-  container: {
```

Comment:
It looks like Appeals `container` does something different than on the other tools pages. Let's not use `ToolsGridContainer` on this page. You can just revert your changes. Sorry I didn't catch that earlier.